src/viking.js

```diff
@@ -1,11 +1,102 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        this.damage = damage;
+        this.health -= damage
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength);
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
```

Comment:
Since you extended Viking from Soldier, and on your `super` you inherit `health` and `strength`, you don't need to pass their values on the constructor. It could be written like this:

```
class Viking extends Soldier {
  constructor(name, health, strength) {
    super(health, strength);
    this.name = name;
  }
(...)
}
```